### ISSUE TITLE
Demo not workin

### ISSUE THREAD

**Issue by newmesiss** [m00s/angular-photoswipe#12]
I'm testing your demo and this does not work, what happens?

**Comment by m00s** [comment]
Hi @newmesiss, which demo are you referring to?

There was one linked in the readme, but I removed it recently because it was outdated.
You can find a working demo in the _demo/ng_ folder.
If that one is not working for you, can you post the errors you are having? Maybe I can help more..

**Comment by raoabishay** [comment]
Hi @m00s , I'm trying to run this within an Ionic application as per the latest demo under demo/ng folder and I cannot get it to work! The DOM element for 'pwsp' appears but it does not show the images and neither does the gallery open. Strangely, I see no errors in the console either! Could you help me set this up? Many thanks in advance!

**Comment by m00s** [comment]
@raoabishay I just tested it again and it is working for me in all major browsers.

Problems could be:
- Demo images are loaded from [lorempixel](http://lorempixel.com), that sometimes is really slow. Try to replace the slides refs in app.js with local images
- The ionic app is somehow blocking the library.
- I noticed that running the demo from the filesystem and the history opt to true as default, the library throws a security error. Try to set `options.history` to `false`

If you still have problem let me know
